templates/message/filter/referent.html.twig

```diff
@@ -11,5 +14,24 @@
             <button type="submit" class="btn btn--dual">Filtrer</button>
             <button type="submit" name="reset" class="btn--as-link b__nudge--left-large text--body text--blue--new">Réinitialiser le filtre</button>
         </div>
-    {{ form_end(form) }}
-</div>
+        {{ form_end(form) }}
+    </div>
+    <div id="message-actions-block" class="l__wrapper--slim background--lighter b__nudge--top-40 text--center"></div>
+{% endblock %}
+
+{% block stylesheets %}
+    {{ parent() }}
+    <link href="{{ asset('select2/select2.min.css') }}" rel="stylesheet">
+{% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+    <script type="text/javascript" src={{ asset('bundles/sonataadmin/vendor/jquery/dist/jquery.min.js') }}></script>
+    <script type="text/javascript" src={{ asset('select2/select2.min.js') }}></script>
+    <script type="text/javascript" src={{ asset('select2/fr.js') }}></script>
+    <script type="text/javascript">
+        Kernel.onLoad(function() {
+            $('.multi_interests').select2({placeholder: '{{ 'referent.filter.interests'|trans|e('js') }}'});
```

Comment:
I think we need to check if `multi_interests` exists before call `.select2` function